manual/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -87,11 +87,13 @@ We also need to reference that id space in our relationships file i.e. `START_ID
 
 === Linux
 
-Under Unix/Linux/OSX, the command is named `neo4j-import`. Depending on the installation type, the tool is either available globally, or used by executing `./bin/neo4j-import` from inside the installation directory.
+Under Unix/Linux/OSX, the command is named `neo4j-admin import --mode=csv`. Depending on the installation type, the tool
+ is either
+available globally, or used by executing `./bin/neo4j-admin import --mode=csv` from inside the installation directory.
```

Comment:
Our standard for docs is to keep each sentence on its own line to make diffs easier to read.